spec/features/items/user_creates_a_new_item_spec.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../shared_scenarios'
+
+RSpec.feature 'user creates a new item' do
```

Comment:
So, I love `let` (and it's synonym `given`)... but I've actually moved back to the good old Arrange, Act, Assert... which of course leads to longer (more repetitive) tests... but I do find them less brittle. This change in approach was triggered by the thoughtbot article [here](https://thoughtbot.com/blog/lets-not)... while I don't agree with all of the reasons for switching, I haven't missed `let` as much as I thought I would :smile:

Just food for thought, because I don't believe there's anything wrong with these feature specs.

Reply:
In future projects I'll keep this in mind.  For time and consistency, I'll stick to using let and given.